pkgs/applications/virtualization/podman/conmon.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchFromGitHub, pkgconfig, glib }:
+
+with lib;
+
+stdenv.mkDerivation rec {
```

Comment:
Please use here `buildGoPackage` otherwise the result will depend on the go compiler.
You can still overwrite the build phase if needed.